person.rb

```diff
@@ -25,4 +26,9 @@ def correct_name
     super
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] I would kindly suggest you refactor your code to achieve the many-to-many (also has-many-through) relationship. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
a better way of writing that 👇

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```